app/controllers/admin/email_domain_blocks_controller.rb

```diff
@@ -21,6 +21,28 @@ def create
 
       if @email_domain_block.save
         log_action :create, @email_domain_block
+
+        if @email_domain_block.with_dns_records?
+          hostnames = []
+          ips       = []
+
+          Resolv::DNS.open do |dns|
+            dns.timeouts = 1
```

Comment:
This is a pretty low timeout. Far lower than what we use for other requests, in fact, while it could on the contrary be raised since it's somewhat unlikely to be used frequently, and can only be triggered by admins.

Reply:
That's a timeout for the DNS server, we use the same DNS timeout in e-mail MX validator and possibly within the request class as well

Reply:
I know it is a timeout for the DNS server. It's a pretty low timeout, though (well, maybe not *that* bad since it only queries one record type and does not resolve CNAMEs or stuff). The `Request` class has a 5s timeout because 1s was too low in practice.